rdtools/aggregation.py

```diff
@@ -0,0 +1,24 @@
+import pandas as pd
+
+
+def aggregation_insol(normalized_energy, insolation, frequency='D'):
+    '''
+    Insolation weighted aggregation
```

Comment:
The flow we had mapped out at the outset was to aggregate after filtering (which is after normalization). We should all be open to re-evaluating that flow (I am!), but I suggest making that a separate (larger) discussion. I have added some explanation in the example notebook